plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -190,9 +198,8 @@ export interface PaginatedEntitiesInitialRequest {
   fields?: (entity: Entity) => Entity;
   limit?: number;
   filter?: EntityFilter;
-  sortField?: string;
+  sortFields?: EntitySortField[];
```

Comment:
Kinda worried that this is a breaking change in the catalog-client. I wonder what we can do about it. Would this be breaking for an existing consumer? Perhaps we should go with a separate implementation for the entity page if that's more convenient

Reply:
hi! no it isn't. These types were introduced in the target PR, specifically for the new `/v2/entities` endpoint 🙂

Reply:
Oh I sorry about my confusion then, I realised this isn't against master 🤦 It's a breaking change for catalog-client once we merge this towards master but I guess we can discuss how to best do that then :) 

Well in that case this looks good to me 👍